doc/dash/dash-sonic-hld.md

```diff
@@ -127,6 +131,7 @@ Following are the minimal scaling requirements
 | Total active connections      | 32M (Bidirectional)           |
 | Metering Buckets per ENI      | 4000                          |
 | CPS                           | 3M                            |
+| Max PA validation entries     | 4k                            |
```

Comment:
Can we add DASH_TUNNEL / MEMBER scale ?

Reply:
added tentative scale values for DASH_TUNNELs